src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h

```diff
@@ -56,43 +59,49 @@ typedef enum EnKaizokuBodyPart {
     /* 15 */ KAIZOKU_BODYPART_MAX
 } EnKaizokuBodyPart;
 
+typedef enum KaizokuTextOffset {
+    /*  0 */ KAIZOKU_COVERSATION_INTRO_1, // shout before landing
+    /*  1 */ KAIZOKU_COVERSATION_INTRO_2, // after landing
+    /*  2 */ KAIZOKU_COVERSATION_WIN,  // after losing to player
+    /*  3 */ KAIZOKU_COVERSATION_LOSS  // after defeating player 
+} KaizokuTextOffset;
+
 typedef struct EnKaizoku {
     /* 0x000 */ PictoActor picto;
     /* 0x148 */ SkelAnime skelAnime;
     /* 0x18C */ Vec3s jointTable[KAIZOKU_LIMB_MAX];
     /* 0x21C */ Vec3s morphTable[KAIZOKU_LIMB_MAX];
     /* 0x2AC */ EnKaizokuActionFunc actionFunc;
     /* 0x2B0 */ s16 action;
-    /* 0x2B2 */ s16 unk_2B2;
+    /* 0x2B2 */ s16 combatTimer;
     /* 0x2B4 */ s16 lookTimer;
-    /* 0x2B6 */ s16 unk_2B6;
-    /* 0x2B8 */ s16 unk_2B8;
+    /* 0x2B6 */ s16 iceTimer;
+    /* 0x2B8 */ s16 colorFilterTimer; // name taken from oot, damaged flashing red
     /* 0x2BA */ s16 drawDmgEffType;
     /* 0x2BC */ s16 switchFlag;
-    /* 0x2BE */ s16 unk_2BE;
     /* 0x2C0 */ f32 drawDmgEffScale;
     /* 0x2C4 */ f32 drawDmgEffFrozenSteamScale;
-    /* 0x2C8 */ s16 unk_2C8;
-    /* 0x2CA */ s16 unk_2CA;
+    /* 0x2C8 */ s16 textIdOffset; // which of the 4 text states during interaction 
+    /* 0x2CA */ s16 textType;
     /* 0x2CC */ s16 eyeIndex;
     /* 0x2CE */ s16 blinkTimer;
-    /* 0x2D0 */ s16 unk_2D0;
+    /* 0x2D0 */ s16 spinAttackState;
     /* 0x2D2 */ s16 swordState;
     /* 0x2D4 */ s16 exitIndex;
     /* 0x2D6 */ s16 csId;
-    /* 0x2D8 */ u8 unk_2D8;
-    /* 0x2D9 */ u8 unk_2D9;
+    /* 0x2D8 */ u8 dontUpdateSkel;
```

Comment:
Maybe something like `disableAnimations`/`pauseAnimations`/`freezeAnimations` might work a bit cleaner for this one? I think those make it a little more obvious what will happen when you set this to `true`.

Reply:
These sound like actions/commands rather than states tho, animationsDisabled?

Reply:
`animationsDisabled` seems good to me.